devlab/tests/test_group_verification.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+import os
+import yaml
+import unittest
+
+from generate_load import Prerequisites
+
+
+class GroupProcedureVerification(unittest.TestCase):
+    """
+    Testing class for grouping verification.
+    """
+    def setUp(self):
+        """
+        SetUp method.
+        """
+        self.src_cloud = Prerequisites(cloud_prefix='SRC')
+        self.main_folder = os.path.dirname(os.path.dirname(os.getcwd()))
+        # TODO:
```

Comment:
Is there appropriate issue inJira to track this the activity about proper package? If not, please, create.